CHANGELOG.md

```diff
@@ -69,6 +69,8 @@ Main (unreleased)
 
 - Fix identifier on target creation for SNMP v2 integration. (@marctc)
 
+- Avoid a panic with an invalid traces configuration. (@ptodev)
```

Comment:
Can this be changed to better explain what the panic was caused by?

Something like this:

> Tracing: avoid a panic when the required `protocols` field was not set in the `otlp` receiver.

Reply:
Thank you, this is done now. I agree that the new description sounds much better.

Reply:
I added a test for this error message as well.